### ISSUE TITLE
StateDetection property TypeError

### ISSUE THREAD

**Issue by Giuseppetm** [Cuberto/mouse-follower#4]
Hello,
I encountered an issue with your library; the code I used is the most basic one to run an example.

```javascript
import MouseFollower from "mouse-follower";
import gsap from "gsap";

MouseFollower.registerGSAP(gsap);
const cursor = new MouseFollower();
```

Versions with this problem: `1.0.4` to `1.0.6`
`1.0.3` and below work fine.

### Error details
![image](https://user-images.githubusercontent.com/52317197/171049925-26c9c325-07bb-46be-83c2-e1722b761fc8.png)

![image](https://user-images.githubusercontent.com/52317197/171049934-e6bbd846-ebcd-4a73-8356-4ed374ab332b.png)

### Environment details
Edge 101.0.1210.53
React 18.1.0
Gsap 3.10.3 (3.10.4 doesn't solve this)
Sass 1.52.1

**Comment by Drafteed** [comment]
Hello! Thanks for the detailed report! Issue should be fixed in v1.0.7

**Comment by Giuseppetm** [comment]
> Hello! Thanks for the detailed report! Issue should be fixed in v1.0.7

Yep, I confirm it's fixed.
Thank you very much! 👍🏻
